### ISSUE TITLE
[DX] Convert /node to a View

### ISSUE THREAD

**Issue by quicksketch** [backdrop/backdrop-issues#162]
Sub-issue of https://github.com/backdrop/backdrop-issues/issues/151. We should convert the main listing provided by node module to a view.

Drupal.org issue: https://drupal.org/node/1806334

**Comment by tarekdj** [comment]
PR: https://github.com/backdrop/backdrop/issues/656
Only 1 test is failing on Travis CI but passes on my local env. Any idea?

**Comment by klonos** [comment]
Is this issue here still relevant? `/node` currently redirects you to the home page, and AFIK we have a layout and a default view for it.

**Comment by laryn** [comment]
@klonos You're right. See @docwilmot comment in the old PR: https://github.com/backdrop/backdrop/pull/656#issuecomment-390415933

**Comment by klonos** [comment]
...yeah, lets close it then 😄

**Comment by klonos** [comment]
While testing the PR for #3343, we've accidentally discovered that this was never actually done. The `promoted` view provided by Backdrop core only provides a block and a feed display, whereas there should be a page display as well (which should have `/node` for its path).

We should also adjust `node_page_default()` to use a `views_embed_view()` + call `watchdog_deprecated_function()`, and remove it later in 2.x.
